js/src/wp-seo-network-admin.js

```diff
@@ -0,0 +1,92 @@
+/* global wpseoNetworkAdminGlobalL10n, ajaxurl */
+
+import a11ySpeak from "a11y-speak";
+
+( function( $ ) {
+	"use strict";
+
+	/**
+	 * Displays given settings errors.
+	 *
+	 * @param {object[]} settingsErrors The list of settings error objects.
```

Comment:
`{object[]}` should be `{Object}`.

Reply:
Just double-checking, even if it's an array of objects?